docs/sources/mimir/manage/monitor-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -14,257 +14,30 @@ You can collect logs and metrics from a Mimir or GEM cluster. To set up dashboar
-              target_label: __address__
-            - regex: (.+)
-              replacement: /api/v1/nodes/${1}/proxy/metrics/cadvisor
-              source_labels:
-                - __meta_kubernetes_node_name
-              target_label: __metrics_path__
-            - action: replace
-              regex: ""
-              replacement: k8s-cluster
-              separator: ""
-              source_labels:
-                - cluster
-              target_label: cluster
-          scheme: https
-          tls_config:
-            ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-            insecure_skip_verify: false
-            server_name: kubernetes
-        - job_name: integrations/grafana-mimir/metrics
-          kubernetes_sd_configs:
-            - role: pod
-          relabel_configs:
-            - action: keep
-              regex: .*metrics
-              source_labels:
-                - __meta_kubernetes_pod_container_port_name
-            - action: keep
-              regex: mimir-distributed-.*
-              source_labels:
-                - __meta_kubernetes_pod_label_helm_sh_chart
-            - action: replace
-              regex: ""
-              replacement: k8s-cluster
-              separator: ""
-              source_labels:
-                - cluster
-              target_label: cluster
-            - action: replace
-              source_labels:
-                - __meta_kubernetes_namespace
-              target_label: namespace
-            - action: replace
-              source_labels:
-                - __meta_kubernetes_pod_name
-              target_label: pod
-            - action: replace
-              source_labels:
-                - __meta_kubernetes_pod_container_name
-              target_label: container
-            - action: replace
-              separator: ""
-              source_labels:
-                - __meta_kubernetes_pod_label_name
-                - __meta_kubernetes_pod_label_app_kubernetes_io_component
-              target_label: __tmp_component_name
-            - action: replace
-              separator: /
-              source_labels:
-                - __meta_kubernetes_namespace
-                - __tmp_component_name
-              target_label: job
-            - action: replace
-              source_labels:
-                - __meta_kubernetes_pod_node_name
-              target_label: instance
-  global:
-    scrape_interval: 15s
-  wal_directory: /tmp/grafana-agent-wal
-```
+Set up Grafana Alloy to collect logs and metrics from Mimir or GEM. To get started with Grafana Alloy,
+refer to [Get started with Grafana Alloy](https://grafana.com/docs/<ALLOY_VERSION>/latest/get-started). After deploying Alloy, refer to [Collect and forward Prometheus metrics](https://grafana.com/docs/alloy/<ALLOY_VERSION>/tasks/collect-prometheus-metrics/) for instructions on how to configure your Alloy instance to scrape Mimir or GEM.
```

Comment:
Do you want to link into the conceptual information - the current "Get started" target? Or to the setup/install information?

Reply:
I was thinking the conceptual information.